docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,632 @@
+| **key**               | **value**                                       |
++-----------------------+-------------------------------------------------+
+|| jwk                  || The public key of the Wallet                   |
+||                      || instance necessary for the                     |
+||                      || holder binding.                                |
++-----------------------+-------------------------------------------------+
+|| vp_formats_supported || The Verifiable Credential and                  |
+||                      || Verifiable Presentation formats                |
+||                      || supported by the Wallet                        |
+||                      || See paragraph. 8.1 of                          |
+||                      || OpenID for Verifiable Presentations - draft 19.|
++-----------------------+-------------------------------------------------+
+
+Signature
+^^^^^^^^^
+
+The JWT (Wallet Instance Attestation) thus composed is signed using the
+private key of the Wallet Provider present to which the Wallet Provider's
+backend has access.
+
+Non-normative example
+^^^^^^^^^^^^^^^^^^^^^
+Below is an example of Wallet Instance Attestation:
+
+.. code-block:: javascript
+
+  {
+    "alg": "ES256",
+    "kid": "5t5YYpBhN-EgIEEI5iUzr6r0MR02LnVQ0OmekmNKcjY",
+    "trust_chain": [
+      "eyJhbGciOiJFUz...6S0A",
+      "eyJhbGciOiJFUz...jJLA",
+      "eyJhbGciOiJFUz...H9gw",
+    ],
+    "typ": "va+jwt",
+    "x5c": ["MIIBjDCCATGgAwIBAgIUZiFoj7bvmhTQvDQtCOY19fMVq/gwCgYIKoZIzj0EAwIwGzEZMBcGA1UEAwwQd2FsbGV0Lml0YWxpYS5pdDAeFw0yMzA2MTMxNDI1NDRaFw0yNjAzMDkxNDI1NDRaMBsxGTAXBgNVBAMMEHdhbGxldC5pdGFsaWEuaXQwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAASqsmuPcB/8Hnuxs4hGtwEztuvvA5zzKePuUcU9N599SdR9HFg8hoL1PMPpDylP8cnDglDU9qNG8LJSELZz1Oojo1MwUTAdBgNVHQ4EFgQUj5fkXeflngzbABlwT6ra1bOQ11cwHwYDVR0jBBgwFoAUj5fkXeflngzbABlwT6ra1bOQ11cwDwYDVR0TAQH/BAUwAwEB/zAKBggqhkjOPQQDAgNJADBGAiEA+avjz/CVnxq+50iAxKLjyRvK/W9XgCp6cKDXs7Y4tugCIQDMWGYL0adwYKhKitYsk8HRIKFDGOZmqZFI35XFehgKQA=="]
+  }
+  .
+  {
+    "iss": "https://wallet.italia.it",
+    "sub": "https://wallet.italia.it/instance/vbeXJksM45xphtANnCiG6mCyuU4jfGNzopGuKvogg9c",
+    "type": "WalletInstanceAttestation",
+    "policy_uri": "https://wallet.italia.it/privacy_policy",
+    "tos_uri": "https://wallet.italia.it/info_policy",
+    "logo_uri": "https://wallet.italia.it/logo.svg",
+    "asc": "https://wallet.italia.it/LoA/basic",
+    "cnf":
+    {
+      "jwk":
+      {
+        "crv": "P-256",
+        "kty": "EC",
+        "x": "4HNptI-xr2pjyRJKGMnz4WmdnQD_uJSq4R95Nj98b44",
+        "y": "LIZnSB39vFJhYgS3k7jXE4r3-CoGFQwZtPBIRqpNlrg",
+        "kid": "vbeXJksM45xphtANnCiG6mCyuU4jfGNzopGuKvogg9c"
+        }
+      },
+      "vp_formats_supported": {
+        "jwt_vc_json": {
+          "alg_values_supported": ["ES256K", "ES384"],
+        },
+        "jwt_vp_json": {
+          "alg_values_supported": ["ES256K", "EdDSA"],
+        },
```

Comment:
```suggestion
  "authorization_endpoint": "eudiw:",
  "response_types_supported": [
    "vp_token"
  ],
  "vp_formats_supported": {
    "jwt_vp_json": {
      "alg_values_supported": ["ES256"]
    },
    "jwt_vc_json": {
      "alg_values_supported": ["ES256"]
    }
  },
  "request_object_signing_alg_values_supported": [
    "ES256"
  ]
```

Reply:
I disagree here. why must the wallet instance attestation have the `presentation_definition_uri_supported` parameter? 

From OIDC4VP:

> Boolean value specifying whether the Wallet supports the transfer of presentation_definition by reference, with true indicating support. If omitted, the default value is true.

Reply:
Since we dont support It, considering we dont have security considerations for that and also that we looks for definition thepufh a trust mark or the trust chain, anyway attested by a third party